### PR TITLE
Add placeholder country option

### PR DESCRIPTION
**Pull request by pculka**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -142,6 +142,7 @@ The phone number can be split into a country choice and phone number text fields
 This produces the preferred choices of 'Jersey' and 'United Kingdom', and regular choices of 'France' and 'United States'.
 
 By default the `country_choices` is empty, which means all countries are included, as is `preferred_country_choices`.
+The option `country_placeholder` can be specified to create a dummy empty selector on top to be able to remove the phone number from the database using this widget.
```
Comment (thewilkybarkid) [inline README.md]: It doesn't have to be empty, it's a string which *could* be empty.

**Comment by thewilkybarkid** [comment]
Could you look at adding a test for this?

**Comment by pculka** [comment]
is this OK ?

**Comment by thewilkybarkid** [comment]
We need to test that the placeholder is actually set. I'd add a new test that behaves similar to this one, except that the data provider provides the actual placeholder (probably `false`, empty string, non-empty string) and expected value (to match against `$view['country']->vars['placeholder']`).

**Comment by pculka** [comment]
I think this ought to do it

**Comment by pculka** [comment]
any info on when you'll merge this in @thewilkybarkid ?

**Comment by shakaran** [comment]
@xabbuh @thewilkybarkid BUMP this PR has the test, it is ok, it should be merged easily, please check

**Comment by pculka** [comment]
Hi,
I'm implementing this bundle again. And again I'm missing this feature. Can you please merge it?

**Comment by tumbochka** [comment]
I would also like to have this feature merged. For now I had to create fork for this. What should be done to merge the feature?

**Comment by rh389** [comment]
This looks straightforward, well covered with tests. I'm going to be bold and merge it.

cc @thewilkybarkid

**Pull request merged**